### PR TITLE
Offline docs: Don't use `-all` Nextflow download

### PR DESCRIPTION
**Pull request by ewels**
Using the `-all` binary in offline systems no longer works, as we have nf-validation and other custom plugins which are not installed (or even installable).

Ideally would simply point to https://nextflow.io/docs/latest/plugins.html#offline-usage but these docs are currently incorrect. A fix is [merged](https://github.com/nextflow-io/nextflow/pull/4447), but not yet live.

Hopefully can remove the bit about pinning versions in a config if/when [this](https://github.com/nextflow-io/nextflow/pull/4340) is merged.

### CODE REVIEW

**Comment by ewels** [comment]
@nf-core-bot fix linting

File: src/content/docs/usage/offline.md [hunk 1]
```diff
@@ -18,17 +18,13 @@ You will need three things: a working version of [Nextflow](#nextflow), the [pip
...
-Download this file and transfer it to your offline system.
-Run it to install Nextflow (it is a very large _bash_ file). See the [Nextflow installation docs](https://www.nextflow.io/docs/latest/getstarted.html#installation) for more installation details and options.
+We do this by installing it locally on a machine that _does_ have an internet connection, and then transferring to the offline system.
 
-Once installed, you can stop Nextflow from looking for updates online by adding the following environment variable in your `~/.bashrc` file:
-
-```bash
-export NXF_OFFLINE='TRUE'
-```
+- Start by [installing Nextflow locally](https://nextflow.io/docs/latest/getstarted.html#installation) (do _not_ use the `-all` package, as this does not allow the use of custom plugins).
```
Comment (mashehu) [inline src/content/docs/usage/offline.md:23]:
```suggestion
- Start by [installing Nextflow locally](https://nextflow.io/docs/latest/getstarted.html#installation) 

:::warning 
do _not_ use the `-all` package, as this does not allow the use of custom plugins.
:::
```

**Review by mirpedrol — APPROVED** [review]

File: src/content/docs/usage/offline.md [hunk 2]
```diff
@@ -18,17 +18,13 @@ You will need three things: a working version of [Nextflow](#nextflow), the [pip
...
+- Kick off a pipeline locally so that Nextflow fetches the required plugins. It does not need to run to completion.
+- Copy the Nextflow binary and `$HOME/.nextflow` folder to your offline environment.
+- In your Nextflow configuration file, specify each plugin that you downloaded, both name and version, including default plugins. This will prevent Nextflow from trying to download newer versions of plugins.
+- Add the following environment variable in your `~/.bashrc` file: `export NXF_OFFLINE='TRUE'`
```
Comment (mashehu) [inline src/content/docs/usage/offline.md:27]:
```suggestion
- Add the following environment variable in your `~/.bashrc` file: 

```bash title=".bashrc"
export NXF_OFFLINE='TRUE'
```

```

File: src/content/docs/usage/offline.md (hunk 1 above)
Comment (ewels) [reply src/content/docs/usage/offline.md]: Added, but indentation so that it kicks in under the bullet in the list.

**Pull request merged**